packages/manager/apps/octavia-load-balancer/src/app.module.js

```diff
@@ -12,6 +12,7 @@ import ngUiRouterBreadcrumb from '@ovh-ux/ng-ui-router-breadcrumb';
 import ovhManagerAtInternetConfiguration from '@ovh-ux/manager-at-internet-configuration';
 import { registerAtInternet } from '@ovh-ux/ng-shell-tracking';
 import OctaviaLoadBalancer from '../../../modules/octavia-load-balancer/src';
+// import OctaviaLoadBalancer from '@ovh-ux/manager-octavia-load-balancer';
```

Comment:
```suggestion
import OctaviaLoadBalancer from '@ovh-ux/manager-octavia-load-balancer';
```

Reply:
It does not work, waiting for @anooparveti solution